dotCMS/src/integration-test/java/com/dotmarketing/portlets/workflows/business/Actionlet_4.java

```diff
@@ -0,0 +1,37 @@
+package com.dotmarketing.portlets.workflows.business;
+
+import com.dotmarketing.portlets.workflows.actionlet.WorkFlowActionlet;
+import com.dotmarketing.portlets.workflows.model.WorkflowActionClassParameter;
+import com.dotmarketing.portlets.workflows.model.WorkflowActionFailureException;
+import com.dotmarketing.portlets.workflows.model.WorkflowActionletParameter;
+import com.dotmarketing.portlets.workflows.model.WorkflowProcessor;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * WorkFlowActionlet use on {@link WorkflowFactoryTest#notCountUniqueSubActionsFromArchivedSchema()}
+ * NOT USE IT ON ANOTHER TEST
```

Comment:
DO NOT*

Reply:
done https://github.com/dotCMS/core/pull/26869/commits/e997967ea222c2992306f6c3c25dc24d2c006b53#diff-26a16a2d3752b681d20e21a3db497e40e98a7d588bd24d6104ce1b3c85b3d55dR14